browser/module.go

```diff
@@ -69,3 +76,459 @@ func (*RootModule) NewModuleInstance(vu k6modules.VU) k6modules.Instance {
+		},
+		"parentFrame": func() *goja.Object {
+			mf := mapFrame(rt, f.ParentFrame())
+			return rt.ToValue(mf).ToObject(rt)
+		},
+		"press":             f.Press,
+		"selectOption":      f.SelectOption,
+		"setContent":        f.SetContent,
+		"setInputFiles":     f.SetInputFiles,
+		"tap":               f.Tap,
+		"textContent":       f.TextContent,
+		"title":             f.Title,
+		"type":              f.Type,
+		"uncheck":           f.Uncheck,
+		"uRL":               f.URL,
+		"waitForFunction":   f.WaitForFunction,
+		"waitForLoadState":  f.WaitForLoadState,
+		"waitForNavigation": f.WaitForNavigation,
+		"waitForSelector": func(selector string, opts goja.Value) *goja.Object {
+			eh := f.WaitForSelector(selector, opts)
+			ehm := mapElementHandle(rt, eh)
+			return rt.ToValue(ehm).ToObject(rt)
+		},
+		"waitForTimeout": f.WaitForTimeout,
+	}
+
+	return maps
+}
+
+//nolint:funlen
+func mapElementHandle(rt *goja.Runtime, eh api.ElementHandle) mapping {
+	maps := mapping{
+		"asElement": func() *goja.Object {
+			m := mapElementHandle(rt, eh.AsElement())
+			return rt.ToValue(m).ToObject(rt)
+		},
+		"dispose":        eh.Dispose,
+		"evaluate":       eh.Evaluate,
+		"evaluateHandle": eh.EvaluateHandle,
+		"getProperties":  eh.GetProperties,
+		"getProperty":    eh.GetProperty,
+		"jSONValue":      eh.JSONValue,
+		"objectID":       eh.ObjectID,
+		"boundingBox":    eh.BoundingBox,
+		"check":          eh.Check,
+		"click":          eh.Click,
+		"contentFrame": func() *goja.Object {
+			f := eh.ContentFrame()
+			mf := mapFrame(rt, f)
+			return rt.ToValue(mf).ToObject(rt)
+		},
+		"dblclick":      eh.Dblclick,
+		"dispatchEvent": eh.DispatchEvent,
+		"fill":          eh.Fill,
+		"focus":         eh.Focus,
+		"getAttribute":  eh.GetAttribute,
+		"hover":         eh.Hover,
+		"innerHTML":     eh.InnerHTML,
+		"innerText":     eh.InnerText,
+		"inputValue":    eh.InputValue,
+		"isChecked":     eh.IsChecked,
+		"isDisabled":    eh.IsDisabled,
+		"isEditable":    eh.IsEditable,
+		"isEnabled":     eh.IsEnabled,
+		"isHidden":      eh.IsHidden,
+		"isVisible":     eh.IsVisible,
+		"ownerFrame": func() *goja.Object {
+			f := eh.OwnerFrame()
+			mf := mapFrame(rt, f)
+			return rt.ToValue(mf).ToObject(rt)
+		},
+		"press":                  eh.Press,
+		"screenshot":             eh.Screenshot,
+		"scrollIntoViewIfNeeded": eh.ScrollIntoViewIfNeeded,
+		"selectOption":           eh.SelectOption,
+		"selectText":             eh.SelectText,
+		"setInputFiles":          eh.SetInputFiles,
+		"tap":                    eh.Tap,
+		"textContent":            eh.TextContent,
+		"type":                   eh.Type,
+		"uncheck":                eh.Uncheck,
+		"waitForElementState":    eh.WaitForElementState,
+		"waitForSelector": func(selector string, opts goja.Value) *goja.Object {
+			eh := eh.WaitForSelector(selector, opts)
+			ehm := mapElementHandle(rt, eh)
+			return rt.ToValue(ehm).ToObject(rt)
+		},
+		"query": func(selector string) *goja.Object {
```

Comment:
Do you want to have those as well?

Reply:
No, nice catch again, thanks, removed in https://github.com/grafana/xk6-browser/pull/673/commits/0c388d50abb43a43f1ab0c6c88e6f274839b47f6.